go/cli/internal/config/config.go

```diff
@@ -40,6 +41,7 @@ func Init() error {
 	viper.SetDefault("user_id", "admin@kagent.dev")
 	viper.SetDefault("output_format", "table")
 	viper.SetDefault("namespace", "kagent")
+	viper.SetDefault("a2a_url", "http://localhost:8083/api/a2a")
```

Comment:
Same question about a new port

Reply:
Ohhh, because the CLI uses autogen directly right now. Crap that's so confusing.